src/pages/Items.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from "react";
+import heartIcon from "../assets/ic_heart.png";
+import "../styles/items.css";
+
+import ItemTop from "../components/ItemTop";
+import Pagnation from "../components/Pagination";
+import useMediaQuery from "../utils/useQueryMedia";
+
+const filterName = {
+  recent: "최신순",
+  favorite: "좋아요순",
+};
+
+const getItems = ({ page, pageSize, orderBy, keyword }) => {
+  return fetch(
+    `https://panda-market-api.vercel.app/products?page=${page}&pageSize=${pageSize}&orderBy=${orderBy}&keyword=${keyword}`
+  ).then((response) => response.json());
+};
+
+function Items() {
+  const isMobile = useMediaQuery("(max-width: 767px)");
+  const isTablet = useMediaQuery("(max-width: 1199px)");
+  const [bests, setBests] = useState([]);
+  const [sells, setSells] = useState([]);
+  const [total, setTotal] = useState(0);
+  const [page, setPage] = useState(1);
+  const [filter, setFilter] = useState("recent");
+
+  useEffect(() => {
+    async function getFavoriteProducts() {
+      const data = await getItems({
+        page: 1,
+        pageSize: 4,
+        orderBy: "favorite",
+        keyword: 0,
+      });
+      setBests(data.list);
+    }
+    getFavoriteProducts();
+  }, []);
+
+  useEffect(() => {
+    async function getRecentProducts() {
+      const data = await getItems({
+        page: page,
+        pageSize: isMobile ? 4 : isTablet ? 6 : 10,
+        orderBy: filter,
+        keyword: 0,
+      });
+      setSells(data.list);
+      setTotal(data.totalCount);
+    }
+    getRecentProducts();
+  }, [filter, page, isMobile, isTablet]);
+
+  return (
+    <>
+      <main className="main-container">
+        <div className="items">
+          <ItemTop>
+            <ItemTop.H1>베스트 상품</ItemTop.H1>
+          </ItemTop>
+          <ul className="best__item-list">
+            {bests.map((item, i) => (
```

Comment:
```suggestion
            {bests.map((item, i) => {
                  const {id, images, description, price, favoriteCount} = item
            return ( 
            <li className="best__item" key={id}>
            ...
            ...
            <p>
                    {description.length > 10
                      ? idescription.slice(0, 10) + "..."
                      : description}
                  </p>
                <strong>{`${price}원`}</strong>
                ...
                ...
            )}

```

소소하지만 이렇게 구조분해할당하면 depth가 하나 줄어듭니다 :-)

Reply:
구조분해할당을 깜빡했네요 수정해야겠습니다